flutter/lib/src/sentry_flutter.dart

```diff
@@ -45,27 +44,26 @@ mixin SentryFlutter {
     final native = SentryNative();
     native.setNativeChannel(nativeChannel);
 
+    final wrapper = PlatformDispatcherWrapper(PlatformDispatcher.instance);
+    final isOnErrorSupported = wrapper.isOnErrorSupported(flutterOptions);
```

Comment:
Since the check is already being done here, we don't need to check the availability again inside the `OnErrorIntegration` class.